Angular InstantSearch/debounced-search-box/package.json

```diff
@@ -45,6 +45,7 @@
     "karma-jasmine-html-reporter": "0.2.2",
     "protractor": "6.0.0",
     "ts-node": "8.1.0",
-    "tslint": "5.16.0"
+    "tslint": "5.16.0",
+    "typescript": "2.7.2"
```

Comment:
why such a low version of typescript?

Reply:
I didn't change that. Yarn reordered the deps.